lib/gui/app/components/details-modal/details-modal.jsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const React = require('react')
+const ReactDOM = require('react-dom')
+const propTypes = require('prop-types')
+const styled = require('styled-components').default
+const { Provider, Button, Modal, Flex, Txt, Box } = require('rendition')
+
+const middleEllipsis = require('./../../utils/middle-ellipsis')
+
+const shared = require('/./../../../../../lib/shared/units')
+const { colors } = require('./../../theme')
+
+const ModalHeader = styled(Flex) `
+  text-align: left;
+  align-items: baseline;
+  font-size: 12px;
+  color: ${colors.light.soft.foreground};
+  padding: 11px 20px;
+  border-bottom: 1.5px solid ${colors.light.soft.background};
+`
+
+const ModalTitle = styled(Txt) `
```

Comment:
You could use a `Box` element with the flex attribute applied here instead of creating a new styled-component